### DISCUSSION TITLE
weird function I found

### DISCUSSION THREAD

**Discussion by dragoncoder047** [kaplayjs/kaplay#387] [General]
why is this function in gfx/anchor.ts?

https://github.com/kaplayjs/kaplay/blob/bd4b9b8c9436d875846f179921e4d7c4ba4988ee/src/gfx/anchor.ts#L44-L46

IMO it should be in a file under audio/ ?

**Comment by lajbel** [comment]
yeas, probably. In the starting time it was all in one file so I moved many stuff without knowing it at all. Thanks for reporting and be active improving KAPLAY!
